### PR TITLE
doc: update ElasticQuota's max value

### PR DESCRIPTION
**Pull request by Gekko0114**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Updated the explanation of ElasticQuota's max value.

#### Special notes for your reviewer:
Thank you for reviewing this PR.

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

**Comment by ffromani** [comment]
/ok-to-test

Hi! Thanks for your contribution!
I for myself do believe that clarifying that "infinite" actually means "maxint" is a useful and welcome addition, but I'd just link back to the golang spec rather than inline the actual value in our docs. Let's see what other reviewers think though.

**Comment by Gekko0114** [comment]
Thanks for your comment !

> but I'd just link back to the golang spec 

Do you mean that I should simply paste the link like this spec?
https://cs.opensource.google/go/go/+/master:src/runtime/string.go;l=354?q=maxint64&ss=go%2Fgo
If I have misunderstood, please let me know!

File: kep/9-capacity-scheduling/README.md [hunk 1]
```diff
@@ -202,7 +202,8 @@ spec:
 ```
 
 The definitions of "min" and "max" have been explained above. Max is infinite
-by default, min is 0 by default. The sum of min in all ElasticQuota must be 
+(math.MaxInt64: 9,223,372,036,854,775,807) by default, min is 0 by default.
```
Comment (Huang-Wei) [inline kep/9-capacity-scheduling/README.md:205]: Readers may not be interested in the concrete number, so:

```suggestion
(in particular `math.MaxInt64`) by default, min is 0 by default.
```

Comment (ffromani) [reply kep/9-capacity-scheduling/README.md:205]: @Gekko0114 (sorry for late answer) this is exactly what I tried to explain previously - sorry if I expressed myself poorly

Comment (Gekko0114) [reply kep/9-capacity-scheduling/README.md]: Thank you so much! I fixed it

**Pull request merged**